test/datahike/backward_compatibility_test/backward_compatibility_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+
+# Run by being in the dir where this file is defined.
```

Comment:
Could you adjust that to run on the root folder so we can include that into our pipelines?

Reply:
Done.